js/app/src/api_docs/ApiDocs.svelte

```diff
@@ -335,4 +352,12 @@
 		padding: var(--size-3);
 		padding-top: 0;
 	}
+
+	a {
+		text-decoration: underline;
+	}
+
+	p.padded {
```

Comment:
The font sizes are really off: 

<img width="934" alt="Screen Shot 2024-02-05 at 11 38 17 AM" src="https://github.com/gradio-app/gradio/assets/9021060/7f0643c8-3c35-4193-8c8f-376bd72a302c">

The text looks really small. and the code snippets for the libraries are too large with too much padding on the bottom. it's weird. 

I suggest adding: 

```
p {
		font-size: var(--text-lg);
	}
```
and modifying the .library css to be 
```
.library {
		border: 1px solid var(--border-color-accent);
		border-radius: var(--radius-sm);
		background: var(--color-accent-soft);
		padding: var(--size-1);
		color: var(--color-accent);
		font-size: var(--text-md);
	}
```

looks much better:

![Screen Shot 2024-02-05 at 11 45 28 AM](https://github.com/gradio-app/gradio/assets/9021060/f4766ae1-b4d0-40ce-bff4-a4beced3f86f)